generated/nidmm/nidmm_service.cpp

```diff
@@ -1824,7 +1824,7 @@ namespace nidmm_grpc {
       };
       uint32_t session_id = 0;
       const std::string& session_name = request->session_name();
-      auto cleanup_lambda = [&] (uint32_t id) { library_->Close(id); };
+        auto cleanup_lambda = [&] (uint32_t id) { library_->Close(id); };
```

Comment:
It looks like fixing the indentation of the `}` on line 39 broke the indentation here.

Reply:
Fixed